src/Scaffolding/VS.Web.CG.Mvc/Templates/RazorPageGenerator/Bootstrap4/CreatePageModel.cshtml

```diff
@@ -52,7 +52,7 @@ namespace @Model.NamespaceName
         [BindProperty]
         public @Model.ViewDataTypeShortName @Model.ViewDataTypeShortName { get; set; }
 
-        // To protect from overposting attacks, see https://aka.ms/RazorPagesCRUD
+        // To protect from overposting attacks, see https://learn.microsoft.com/aspnet/core/blazor/forms/#mitigate-overposting-attacks
```

Comment:
Revert this back to the `aka.ms` link. The content for the Blazor link only applies to Razor components.

```suggestion
        // To protect from overposting attacks, see https://aka.ms/RazorPagesCRUD.
```

Reply:
sg fixed